apps/acrs/behaviours/aggregator-save-update.js

```diff
@@ -23,6 +25,8 @@ module.exports = superclass => class extends superclass {
 
   updateItem(req, res) {
     const id = req.params.id;
+    updatingIndex = id;
+    isUpdate = true;
```

Comment:
`isUpdate` is never removed or reset and being in the global scope this means that after the first edit isUpdate seems to always be true and subsequent 'adds' are spliced in ignoring the aggregateLimit check

Reply:
Not sure it does that but once a major events are triggered all variable are always set to default, i was able to add more member after edit. i have unset after the action is completed